src/components/Header/MobileMenu.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react"
+import { useSelector, useDispatch } from "react-redux"
+import { NavLink } from 'react-router-dom'
+
+import { fetchLogout } from "../../ducks/userInfo"
+import LogoHead from "./LogoHead"
+import PointsBal from "./PointsBal"
+import Navigation from "./Navigation"
+import {
+    ContainerHeader,
+    ExitIconWrapper,
+    LeftBlock,
+    MenuHeaderBlock,
+    MenuInitiaLetterCircle,
+    MenuInitiaLetter,
+    MenuName,
+    MobileHamburgerContainer,
+    MobileHamburgerLine,
+    MobileHamburgerbackground,
+    MobileHamburgerLoginBox
+} from "./Header.styled"
+import ExitIconMobile from "../../assets/image/Header-exit-icon-mobile.svg"
+
+const MobileMenu = () => {
+    let userName = 'email'
```

Comment:
const userName = userData ? userData.email ? null;
const nameLetter = userName ? userName.charAt(0) : null

Reply:
Thanks for the professional advice)) Everything is fixed

![image](https://user-images.githubusercontent.com/96341034/189455139-e71a70fc-3f99-47fe-af5d-2312faed6a16.png)